proteus

```diff
@@ -148,7 +148,13 @@ class WriteComposeFile:
         files = glob.glob("/dev/xclmgmt*")
         for file in files:
             devices.append(f"{file}:{file}")
-
+        # migraphx devices, if found
+        kfd = glob.glob('/dev/kfd')
+        dri = glob.glob('/dev/dri')
+        if len(kfd) == 1:
+            devices.append(kfd)
+        # if len(dri) == 1:
+        devices.append(dri)
```

Comment:
glob returns an array. You should be iterating through the array and adding its contents like the existing devices